go.mod

```diff
@@ -24,7 +24,7 @@ require (
 	github.com/zclconf/go-cty v1.15.0
 	go.mongodb.org/atlas v0.36.0
 	go.mongodb.org/atlas-sdk/v20240530005 v20240530005.0.0
-	go.mongodb.org/atlas-sdk/v20240805001 v20240805001.0.1-0.20240813143344-1cfe40386cbc
+	go.mongodb.org/atlas-sdk/v20240805001 v20240805001.1.1-0.20240826083604-5148b5e06fe3
```

Comment:
Had to update dev preview to pick up latest changes in list operation supporting pagination.